db/data_migration/20160527170207_share_content_ids_for_similar_html_attachments.rb

```diff
@@ -0,0 +1,12 @@
+HtmlAttachment.connection.execute(<<-SQL)
```

Comment:
I've seen this in `html_publication_sync_check.rb` as well, so I assume it's valid, but can you really close the argument paranthesis and then follow on the next line with the actual SQL?